xds/src/test/java/io/grpc/xds/XdsNamResolverTest.java

```diff
@@ -66,6 +69,23 @@ public void uncaughtException(Thread t, Throwable e) {
           .build();
 
   private final XdsNameResolverProvider provider = new XdsNameResolverProvider();
+  private static final Node bootstrapNode = Node.newBuilder().build();
+  private static final Bootstrapper bootstrapper = new Bootstrapper() {
+    @Override
+    String getServerUri() {
+      return "fake_server_uri";
+    }
+
+    @Override
+    Node getNode() {
+      return bootstrapNode;
```

Comment:
Use `Node.getDefaultInstance()`.

Reply:
We should not use default instance for testing.